src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,24 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+interface ListProps {
```

Comment:
```suggestion
interface TodoInfoProps {
```
lets be more descriptive :)